spec/system/users/sign_up_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+feature 'User sign up' do
+  let(:email) { Faker::Internet.email }
+  let(:password) { Faker::Internet.password(min_length: 8) }
+
+  feature 'with correct data' do
+    scenario 'signs up successfully' do
+      visit new_user_registration_path
+
+      fill_in 'Email', with: email
+      fill_in 'Password', with: password
+      fill_in 'Password confirmation', with: password
```

Comment:
Page objects? 😬

Reply:
I like them, maybe we can leave it for another PR, thoughts?

Reply:
Love the idea, the more we can split the better